src/unified-namespace/index.njk

```diff
@@ -0,0 +1,182 @@
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="nohero w-full ff-bg-light">
+    <!-- Learn UNS -->
+    <div class="about w-full bg-white pt-12 md:pt-12 pb-12">
+        <div class="text-center">
+            <h2>Learn about the Unified Namespace (UNS)</h2>
+        </div>
+        <div class="container m-auto max-w-4xl">
+            <div
+                class="grid grid-cols-1 md:grid-cols-2 pt-4 md:pt-6 gap-x-8 items-start">
+                <!-- Quick Tips Column -->
+                <div class="px-6 pb-6 md:pb-0">
+                    <h3 class="w-full text-center mb-4">Getting Started</h3>
+                    <ul class="ff-nodered-resources">
+                        <a href="/blog/2023/12/introduction-to-unified-namespace/">
+                            <li>
+                                <div class="ff-nodered-resources--icon">{% ffIconLg "nodered",
+                                    true %}</div>
+                                <label>Introduction into the UNS</label>
+                            </li>
+                        </a>
+                        <a href="/blog/2023/08/isa-95-automation-pyramid-to-unified-namespace/">
+                            <li>
+                                <div class="ff-nodered-resources--icon">{% ffIconLg "bug",
+                                    true %}</div>
+                                <label>From ISA 95 towards the UNS</label>
+                            </li>
+                        </a>
+                    </ul>
+                </div>
+                <!-- Concepts Column -->
+                <div class="px-6 pb-6 md:pb-0">
+                    <h3 class="w-full text-center mb-4">Implementation of the UNS</h3>
+                    <ul class="ff-nodered-resources">
+                        <a href="/blog/2023/12/unified-namespace-data-modelling/">
+                            <li>
+                                <div class="ff-nodered-resources--icon">{% ffIconLg "chart",
+                                    true %}</div>
+                                <label>Comparing Node-RED Dashboard Solutions</label>
+                            </li>
+                        </a>
+                        <a href="/blog/2024/01/unified-namespace-what-broker/">
+                            <li>
+                                <div class="ff-nodered-resources--icon">{% ffIconLg "lock",
+                                    true %}</div>
+                                <label>Selecting your UNS Broker</label>
+                            </li>
+                        </a>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="text-white">
+        {% include "components/dividers/light-grey-curve-inverted.svg" %}
+    </div>
+    <!-- Frequently Asked Questions -->
```

Comment:
This section is not showing the answers, you need the script that's in the Node-RED page to make the accordions work.

Reply:
I created a partial an added it to the node red page and to this one as well